server/src/utils/fight/getFighters.ts

```diff
@@ -93,6 +95,22 @@ const handleSkills = (brute: Brute, fighter: DetailedFighter) => {
   }
 };
 
+const handleMapModifiers = (
+  brute: Brute,
+  effect: MapEffect,
+) => { 
+  switch(effect){
+    case MapEffect.None:
+      break;
+    case MapEffect.HalfSpeed:
+      brute.speedValue *= 0.5;
+    case MapEffect.OneHp:
+      brute.hp *= 0.1;
```

Comment:
The modifier OneHp is wrongly named, as it doesn't make the fighters have one HP, but a tenth.
Also, HPs should always be integers, so you need to round the result.
Same for the speed, it needs to be rounded